server/protocols/jmap/src/test/java/org/apache/james/jmap/model/MessageFactoryTest.java

```diff
@@ -361,6 +361,32 @@ public void dateFromHeaderShouldBeUsedIfPresent() throws Exception {
     }
 
     @Test
+    public void dateFromHeaderShouldUseCurrentCenturyWhenNone() throws Exception {
```

Comment:
Could you add a test with 99 if we decide that 99 should go to 1999 and not 2099